### PR TITLE
Final version Portofolio

### PR DESCRIPTION
**Pull request by TiteLunda**
Added a Work Section
Added About Section
Styled Work and About Sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+
+    <header>
+
+
+        <nav class="navbar fixed-top">
+            <a href="" id="logo">
+                Welcome
+              </a>
+
+            <img src="/Images/Union.png" alt="navigation bars" class="menu">
```
Comment (joel-kalema) [inline index.html]: Good job working on the mobile version 😃, however, you need to fix some part

- [ ] Kindly, make sure you stuck to the design as much as possible (`font`, `colors`, `images`, `text`, `margins`) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=7V3y8z3F5uwwsjzJ-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍. As you can see the link images are broken also 😃. 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |Capture0| |Capture| |

File: index.html [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            <hr class="line ">
+        </div>
+
+        <div class="cards ">
+            <div class="cardsContent ">
+                <img src="./Images/Img Plaholder.png " alt="Store image ">
+                <h3 class="title1 ">Multi-Post Stories Gain+Glory</h3>
+                <ul class="Languages ">
+                    <li class="language ">Ruby on rails </li>
+                    <li class="language ">Css </li>
```
Comment (joel-kalema) [inline index.html]: - [ ] Kindly, make sure you fixed your `margin` to stick to the design as much as possible using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48-988&t=7V3y8z3F5uwwsjzJ-0) as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md#:~:text=You%20must%20stick%20to%20the%20design%20as%20much%20as%20possible%20(e.g.%2C%20font%2C%20colors%2C%20images%2C%20text%2C%20margins)%20using%20the%20templates%20in%20Figma.) 👍

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |Capture| |Capture0| |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @TiteLunda 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by braincee — APPROVED** [review]
Hi @TiteLunda, I am @braincee, your Code Reviewer.

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

youre-approved-shirley-auto-group

OPTIONAL CHANGES
---

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @braincee._

---

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**